librocksdb-sys/build.rs

```diff
@@ -90,6 +90,18 @@ fn build_rocksdb() {
         .filter(|file| *file != "util/build_version.cc")
         .collect::<Vec<&'static str>>();
 
+    if cfg!(target_arch = "x86_64") {
+        // This is needed to enable hardware CRC32C. Technically, SSE 4.2 is
+        // only available since Intel Nehalem (about 2010) and AMD Bulldozer
+        // (about 2011).
+        config.define("HAVE_PCLMUL", Some("1"));
+        config.define("HAVE_SSE42", Some("1"));
+        config.flag("-msse2");
```

Comment:
Is there any reason to use `.flag_if_supported()` here or it's redundant and every cpu with arch `x86_64` supports these instructions ?